Testing/ORKTest/ORKTest/MainViewController.m

```diff
@@ -2945,7 +2984,14 @@ Tests customizing continue button ("some of the time").
         [items addObject:@"Item2"];
         [items addObject:@"Item3"];
         stepViewController.navigationItem.titleView = [[UISegmentedControl alloc] initWithItems:items];
+    }else if ([stepViewController.step.identifier isEqualToString:@"waitTask.step2"]) {
+        // Indeterminate step
+        [((ORKWaitStepViewController *)stepViewController) performSelector:@selector(finish) withObject:nil afterDelay:5.0];
```

Comment:
The call to `finish` seems to be the only reason as to why an `ORKActiveStep` is used. We should convert it from an `ORKActiveStep` and `ORKActiveStepViewController` to `ORKStep` and `ORKStepViewController`. Then you can just call `[self goForward]` which is available as part of `ORKStepViewController`.